.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,5 @@
+# problem statement
+- What does is the problem or feature request.
```

Comment:
Has anyone really been far even as decided to use even go want to do look more like?